server/services/ws-services-requests-manager.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+var requestLogger = require('../../webservice/request-logger')
+  , deferred = require('deferred')
+  , resolveProjectRoot = require('cjs-module/resolve-project-root')
+  , path        = require('path')
+  , projectRoot = process.cwd()
+  , debug       = require('debug-ext')('ws-setrvices-requests-manager');
+
+exports.getRequestsDirectory = function () {
```

Comment:
@kamsi i think this should be memoized, we will probably use this service more often than at every restart (to be decided however), while the file structure won t change during app run
(though not super performance booster :))